config/environment_indicator.switcher.local.yml

```diff
@@ -0,0 +1,11 @@
+uuid: 9a67c49d-bdc8-4c97-be28-5e94e65cd424
+langcode: en
+status: true
+dependencies: {  }
+machine: local
+description: null
+name: Local
+weight: ''
+url: 'https://odsg.test/'
+fg_color: '#ffffff'
+bg_color: '#0d0d0d'
```

Comment:
This isn't quite black. not sure if that's on purpose.

Reply:
Nope, not intentional, thanks for picking it up.